src/main/java/org/eclipse/microprofile/starter/rest/API.java

```diff
@@ -0,0 +1,77 @@
+package org.eclipse.microprofile.starter.rest;
+
+import org.eclipse.microprofile.starter.addon.microprofile.servers.model.MicroprofileSpec;
+import org.eclipse.microprofile.starter.addon.microprofile.servers.model.SupportedServer;
+import org.eclipse.microprofile.starter.core.model.JavaSEVersion;
+import org.eclipse.microprofile.starter.core.model.MicroProfileVersion;
+import org.eclipse.microprofile.starter.rest.model.Project;
+
+import javax.inject.Inject;
+import javax.validation.constraints.NotNull;
+import javax.ws.rs.ApplicationPath;
+import javax.ws.rs.Consumes;
+import javax.ws.rs.GET;
+import javax.ws.rs.HeaderParam;
+import javax.ws.rs.POST;
+import javax.ws.rs.Path;
+import javax.ws.rs.PathParam;
+import javax.ws.rs.Produces;
+import javax.ws.rs.QueryParam;
+import javax.ws.rs.core.Application;
+import javax.ws.rs.core.HttpHeaders;
+import javax.ws.rs.core.Response;
+import java.util.List;
+
+/**
+ * @author Michal Karm Babacek <karm@redhat.com>
+ */
+@ApplicationPath("/api")
+@Path("/1.0.0")
+public class API extends Application {
```

Comment:
Uncommon to define a JAX-RS resource within the Application class. Separate this into 2 classes.

Reply:
Fixed.